app/components/Exchange/MyMarkets.jsx

```diff
@@ -968,7 +970,9 @@ class MyMarkets extends React.Component {
                         </div>
                         <div className="search-wrapper">
                             <form>
-                                <input
+                            <div className="filter inline-block">
+
+                                <Input
```

Comment:
The new component seems to have unnecessary margins, what is causign that?
![image](https://user-images.githubusercontent.com/33128181/49017980-71b0e780-f18a-11e8-95d6-02707a1c0d0b.png)

Reply:
removed padding and some margin